### PR TITLE
Improved srfi-tools Chibi Scheme support.

### PR DESCRIPTION
**Pull request by shawnw**
* Mention the required dependency on SRFI-19 and how to install it.
* Add make-chibi-image.sh script to generate an image with all the srfi-tools modules preloaded, and adjust wrapper/chibi-scheme to use it if present.

### CODE REVIEW

File: srfi-tools/make-chibi-image.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
```
Comment (lassik) [inline srfi-tools/make-chibi-image.sh:2]: `cd "$(dirname "$0")"` to change to the directory where the script it. Once you're in that directory, you can `cd ..` and so on.

**Comment by shawnw** [comment]
I'm now thinking about replacing that script with a Makefile for running
any scheme-specific setup, so you can just do `make chicken`, `make chibi`,
etc. instead.

**Comment by lassik** [comment]
Fine by me.

**Comment by lockywolf** [comment]
Useful contribution!

**Comment by arthurgleckler** [comment]
Shawn, thanks for this.  I'll wait to merge anything until you give the word.

**Comment by shawnw** [comment]
> Shawn, thanks for this. I'll wait to merge anything until you give the word.

Right now I'm waiting on a bug with Chibi's SRFI-193 implementation that's preventing image files that include that module from being dumped to be fixed so I can confirm it works with the latest build, not just the 0.10.0 release.

**Comment by lockywolf** [comment]
@shawnw Fixed?

**Comment by shawnw** [comment]
> @shawnw Fixed?

Yeah, the issue's been fixed for real in chibi (If you're using a git version you'll want to update). Switched to using a Makefile to build the image file.

@arthurgleckler Should be good to merge now.

**Comment by arthurgleckler** [comment]
Thanks.  Merged and pushed.  And it works for me.

**Comment by lassik** [comment]
Works for me too.

Perhaps the image wrapper should be separate from the non-image wrapper? If you work on the code, it will be slow to make a new image after each change. And the image can work as a "last known good version" that you can run side by side with the new experimental code.

**Comment by shawnw** [comment]
> Works for me too.
> 
> Perhaps the image wrapper should be separate from the non-image wrapper? If you work on the code, it will be slow to make a new image after each change. And the image can work as a "last known good version" that you can run side by side with the new experimental code.

I tend towards a 'If you're actively working on the srfi-tools code and using chibi you'll presumably not use the wrapper during development and can easily bypass using an existing image file manually when desired' school of thought. Which, granted, very well might be a minority opinion.

**Pull request merged**